### PR TITLE
add organization activity chart

### PR DESCRIPTION
**Pull request by larsxschneider**
Calculate the number of active organizations for the last day, week,
and 4 weeks. Visualize the numbers in a new chart.

The naming is a bit inconsistent (`org` vs `orgs` vs `organization`). As always, I am open for suggestions!

### CODE REVIEW

File: updater/reports/ReportOrgActivity.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+					users.id
+				FROM
+					repositories
+					JOIN users ON repositories.owner_id = users.id
+					JOIN pushes ON pushes.repository_id = repositories.id
+				WHERE
+					users.type = "Organization" AND
+					CAST(pushes.created_at AS DATE) BETWEEN "''' + str(timeRange[0]) + '''" AND "''' + str(timeRange[1]) + '''"'''
+
+		query +=   self.andExcludedEntities("users.login")
```
Comment (pluehne) [inline updater/reports/ReportOrgActivity.py]: The spacing after `+=` is odd here.

File: docs/demo-data/organization-activity.tsv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+date	last four weeks	last week	last day
+2018-01-23	26	23	18
+2018-01-22	26	22	16
+2018-01-21	26	21	13
+2018-01-20	26	22	16
+2018-01-19	26	23	17
+2018-01-18	26	23	12
```
Comment (pluehne) [inline docs/demo-data/organization-activity.tsv]: Maybe, we could add some more data points?

File: updater/reports/ReportOrgActivity.py (hunk 1 above)
Comment (larsxschneider) [reply updater/reports/ReportOrgActivity.py]: Fixed!

File: docs/demo-data/organization-activity.tsv (hunk 2 above)
Comment (larsxschneider) [reply docs/demo-data/organization-activity.tsv]: I would argue that this is enough to see how the chart would look like. The chart is not that unusual to warrant more data I would say 😄

Comment (pluehne) [reply docs/demo-data/organization-activity.tsv]: Okay, that was just a suggestion anyways 🙂.

File: updater/reports/ReportOrgActivity.py (hunk 1 above)
Comment (pluehne) [reply updater/reports/ReportOrgActivity.py]: Thanks!

**Pull request merged**